dev.zed.Zed-Preview.yaml

```diff
@@ -0,0 +1,194 @@
+      # Modify original `.desktop` file to use the correct icon name
+      - desktop-file-edit --set-icon="${FLATPAK_ID}" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: archive
+        url: https://github.com/zed-industries/zed/releases/download/v0.186.0-pre/zed-linux-x86_64.tar.gz
+        sha256: 4ed95b98a7ec62ef6a75677e858c8924824fcde53fe711d215163850ac07d816
+        only-arches:
+          - x86_64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/zed-industries/zed/releases
+          version-query: map(select(.prerelease)) | first | .tag_name
+          url-query: map(select(.prerelease)) | first | .assets[] | select(.name=="zed-linux-x86_64.tar.gz")
+            | .browser_download_url
+          is-main-source: true
+      - type: archive
+        url: https://github.com/zed-industries/zed/releases/download/v0.186.0-pre/zed-linux-aarch64.tar.gz
+        sha256: de856a2a2f33d46d5b6b1afbeeb0f6a23f66f7da5da9b23661c96557e6fa0f4e
+        only-arches:
+          - aarch64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/zed-industries/zed/releases
+          version-query: map(select(.prerelease)) | first | .tag_name
+          url-query: map(select(.prerelease)) | first | .assets[] | select(.name=="zed-linux-aarch64.tar.gz")
+            | .browser_download_url
+          is-main-source: true
+      - type: file
+        path: dev.zed.Zed-Preview.metainfo.xml
+
+  - name: host-spawn
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 host-spawn /app/bin/host-spawn
+    sources:
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.1/host-spawn-x86_64
+        sha256: 733746ab498e07d065cbecf80bacd447eb21846d1462e8fe23fdd9d9dc977b50
+        dest-filename: host-spawn
+        only-arches: [x86_64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/1player/host-spawn/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test(".+x86_64$"))][0] | .browser_download_url'
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.1/host-spawn-aarch64
+        sha256: 71b7744a4d0b29279523cc0f0ed1a7af5e9555510bd9e6d4685391b59faefaac
+        dest-filename: host-spawn
+        only-arches: [aarch64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/1player/host-spawn/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test(".+aarch64$"))][0] | .browser_download_url'
+
+  - name: zed-wrapper
+    buildsystem: meson
+    config-opts:
+      - -Deditor_binary=/app/bin/zed
+      - -Deditor_title=Zed Preview
```

Comment:
```suggestion
      - -Deditor_title=Zed
```

Reply:
Committed, builds will take some time to run, the queue is overloaded.

Reply:
ok - thanks!